pyanaconda/core/configuration/payload.py

```diff
@@ -41,16 +41,17 @@ def ignored_packages(self):
         return self._get_option("ignored_packages", str).split()
 
     @property
-    def enable_updates(self):
-        """Enable installation of latest updates.
+    def updates_repositories(self):
+        """List of names of repositories that provide latest updates.
 
-        This flag controls whether or not Anaconda should provide an option to
-        install the latest updates during installation source selection.
+        This option also controls whether or not Anaconda should provide
+        an option to install the latest updates during installation source
+        selection.
 
-        The installation of latest updates is selected by default, if the closest
-        mirror is selected, and the "updates" repo is enabled.
+        The installation of latest updates is selected by default, if
+        the closest mirror is selected, and the "updates" repo is enabled.
         """
-        return self._get_option("enable_updates", bool)
```

Comment:
If there are customers using the `enable_updates` option, this would break their configuration.
IMO, `enable_updates` needs to still be supported in RHEL-8. It is also documented in [1], so removing it would have a doc impact.

[1]https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html-single/customizing_anaconda/index#configuring_the_default_configuration_files